mpmt/README.md

```diff
@@ -0,0 +1,78 @@
+   |Gratificação Natalina|Parcelas da Gratificação Natalina (13º) pagas no mês corrente.|
+   |Férias (1/3 Constitucional)|Adicional correspondente a 1/3 (um terço) da remuneração, pago ao membro por ocasião das férias.|
+   |Abono de Permanência|Valor equivalente ao da contribuição previdenciária, devido ao funcionário público que esteja em condição de aposentar-se, mas que optou por continuar em atividade (instituído pela Emenda Constitucional nº 41, de 16 de dezembro de 2003.|
+   |Outras Remunerações Temporárias|Outras remunerações temporárias.|
+   |Verbas Indenizatórias|Auxílio-Alimentação, Auxílio-Transporte, Auxílio-Moradia, Ajuda de Custo e outras desta natureza, exceto diárias, que são divulgadas no Portal da Transparência.|
+   |Total de Rendimentos Brutos|Total dos rendimentos brutos pagos no mês.|
+   |Contribuição Previdenciária|Contribuição Previdenciária Oficial (Plano de Seguridade Social do Servidor Público e Regime Geral de Previdência Social).|
+   |Imposto de Renda|Imposto de Renda Retido na Fonte.|
+   |Retenção por Teto Constitucional|Valor deduzido da remuneração básica bruta, quando esta ultrapassa o teto constitucional, nos termos da legislação correspondente.|
+   |Total de Descontos|Total dos descontos efetuados no mês.|
+   |Rendimento Líquido Total|Rendimento líquido após os descontos referidos nos itens anteriores.|
+
+2. Contracheque - Verbas Indenizatórias
+   |Campo|Descrição|
+   |-|-|
+   |VERBAS INDENIZATÓRIAS|Auxílio-alimentação, Auxílio-transporte, Auxílio-moradia, Ajuda de custo e outras dessa natureza, exceto diárias, que serão divulgadas no Portal da Transparência, discriminada de forma individualizada.|
+   |OUTRAS REMUNERAÇÕES TEMPORÁRIAS|Valores pagos a título de Adicional de Insalubridade ou de Periculosidade, Adicional Noturno, Serviço Extraordinário, Substituição de Função, Cumulações.|
+Estas planilhas contém as informações de pagamento do mês que foi passado como parâmetro, a fim de gerar os *crawling results* de cada mês.
+
+## Como usar
+> Remova o sifrão "$".
+### Executando com Docker
+Para executar com o docker não é necessário passar a pasta de download via parâmetro. 
+ - Inicialmente é preciso instalar o [Docker](https://docs.docker.com/install/). 
+
+ - Construção da imagem:
+
+    ```sh
+    $ cd coletores/mpmt
+    $ docker build -t mpmt .
+    ```
+ - Execução:
+ 
+    ```sh
+    $ docker run -e YEAR=2020 -e MONTH=2 -e GIT_COMMIT=$(git rev-list -1 HEAD) mpmt
+    ```
+### Execução sem o Docker:
+
+- Para executar o script é necessário rodar o seguinte comando, a partir do diretório `/mpmt`, adicionando às variáveis seus respectivos valores, a depender da consulta desejada. É válido lembrar que faz-se necessario ter o [Python 3.6.9](https://www.python.org/downloads/) instalado.
+
+    ```sh
+    $ MONTH=1 YEAR=2018 OUTPUT_FOLDER="Local de Download" GIT_COMMIT=$(git rev-list -1 HEAD) python3 src/main.py
```

Comment:
Coloquei o `OUTPUT_FOLDER`, para quem for usar depois, saber que isso existe, e para mim quando for copiar o comando, se deixar no automático as planilhas vão para o /output, na raiz.